src/Service.Host/client/src/components/requisitions/PickStockDialog.js

```diff
@@ -52,6 +53,16 @@ function PickStockDialog({
         setOpen(false);
     };
 
+    const useStyles = makeStyles(() => ({
```

Comment:
mui/styles is legacy and no longer supported. I think it still works, but mui are moving away from this kind of styling (On my mind since I have spent the day ripping it out of finance)

https://v6.mui.com/system/styles/basics/